dist/src/main/java/io/camunda/application/commons/configuration/BrokerBasedConfiguration.java

```diff
@@ -79,12 +81,23 @@ public SchedulerConfiguration schedulerConfiguration() {
     return new SchedulerConfiguration(cpuThreads, ioThreads, metricsEnabled, "Broker", nodeId);
   }
 
-  @ConditionalOnProperty(prefix = "zeebe.broker.gateway", name = "enable", havingValue = "false")
+  @ConditionalOnExpression(
+      "'${zeebe.broker.gateway.enable:true}' == 'false' or '${camunda.rest.enabled:true}' == 'false'")
```

Comment:
❌ I'm not sure about this, it should be something like:

```suggestion
      "'${zeebe.broker.gateway.enable:false}' or '${camunda.rest.enabled:false}'")
```

Reply:
If I understand the syntax right, we provide the default values if the properties are missing after the `:` and the default value is `true` for both properties in case they are not defined. Alas, the definition should be fine in my regard. Let me know if I overlooked anything.

Reply:
I don't think is the right way to provide the default values. You can also do like:

```suggestion
      @ConditionalOnProperty(
    name = {"zeebe.broker.gateway.enable", "camunda.rest.enabled"},
    havingValue = "false",
    matchIfMissing = true)
```

Reply:
I have simplified the property handling, this should not be relevant anymore.